docs/tutorial/part-one/index.md

```diff
@@ -62,9 +62,10 @@ nothing is happening at first; be patient!
 Now let's try running Gatsby!
 
 Gatsby has a built-in development server. Let's start it up by running the
-following command:
+following commands:
 
 ```sh
+npm install
```

Comment:
Gatsby will automatically install the needed npm packages when using the `gatsby new` command.